cli/lib/utils.js

```diff
@@ -0,0 +1,99 @@
+const path = require('path');
+const fs = require('fs');
+const loadJSON = require('load-json-file');
+const writeJSON = require('write-json-file');
+const ApifyClient = require('apify-client');
+const { error } = require('./outputs');
+const { APIFY_LOCAL_EMULATION_DIR, APIFY_DEFAULT_DATASET_ID,
+    APIFY_DEFAULT_KEY_VALUE_STORE_ID, GLOBAL_CONFIGS_FOLDER,
+    AUTH_FILE_PATH, LOCAL_CONFIG_NAME, APIFY_LOCAL_DATASETS_DIR,
+    APIFY_LOCAL_KEY_VALUE_STORES_DIR } = require('./consts');
+const { createFolderSync, updateLocalJSON } = require('./files');
+
+const getLoggedClientOrError = async () => {
+    if (!fs.existsSync(GLOBAL_CONFIGS_FOLDER) || !fs.existsSync(AUTH_FILE_PATH)) {
+        error('You aren\'t logged call "apify login" to process login.');
+        return;
+    }
+    const auth = loadJSON.sync(AUTH_FILE_PATH);
```

Comment:
Maybe we should catch JSON parse error and display some message about malformed auth file